src/html.htl

```diff
@@ -1,9 +1,23 @@
 <!DOCTYPE html>
 <html>
 <head>
-    <title>${content.title}</title>
-    <meta data-sly-test.hash=${content.data.sourceHash} name="x-source-hash" content="${hash}">
-    <esi:include src="/head.html" onerror="continue"/>
+  <title>${content.title}</title>
+  <meta data-sly-test.hash=${content.data.sourceHash} name="x-source-hash" content="${hash}"/>
```

Comment:
If you're not re-using hash (which it looks like you're not), you don't need to assign an identifier here.

Reply:
Yeah, I saw that but decided not to touch that line (other than for whitespace reduction)...

Reply:
the source hash is added by the pipeline:
https://github.com/adobe/helix-pipeline/blob/0b695119e12ac82efacc298e005f0241b541e15f/src/html/fetch-external.js#L106